STMicroelectronics/SPWF04Sx/SPWF04SxInterface.cs

```diff
@@ -187,6 +187,418 @@ public class SPWF04SxInterface : NetworkInterface, ISocketProvider, ISslStreamPr
             return result.Substring(result.IndexOf(':') + 1);
         }
 
+        //*************************************** Begin Commands added by RoSchmi  ***************************************************
```

Comment:
This type of comment will fall out of date as the driver evolves, same with the matching end one